packages/vuetify/src/components/VBreadcrumbs/VBreadcrumbs.tsx

```diff
@@ -91,24 +99,33 @@ export const VBreadcrumbs = genericComponent<new <T extends BreadcrumbItem>(
     const items = computed(() => props.items.map(item => {
       return typeof item === 'string' ? { item: { title: item }, raw: item } : { item, raw: item }
     }))
+    const showEllipsis = computed(() => items.value.length >= props.maxItems)
+    const enableEllipsis = ref(showEllipsis.value)
+
+    const onClickEllipsis = () => {
+      enableEllipsis.value = false
+    }
 
     useRender(() => {
       const hasPrepend = !!(slots.prepend || props.icon)
 
       return (
         <props.tag
-          class={[
-            'v-breadcrumbs',
-            backgroundColorClasses.value,
-            densityClasses.value,
-            roundedClasses.value,
-            props.class,
-          ]}
-          style={[
-            backgroundColorStyles.value,
-            props.style,
-          ]}
+          aria-label="breadcrumbs"
         >
+          <ol
+            class={[
+              'v-breadcrumbs',
```

Comment:
Is there a benefit to nesting `v-breadcrumbs` + all classes and styles and leaving `<nav>` almost empty? I don't think we have a precedence of this kind. Usually all those things go into the root element.